crossdock/server/endtoend.py

```diff
@@ -65,8 +66,13 @@ def __init__(self):
         init_sampler = cfg.sampler
         channel = self.local_agent_sender
 
+        sender = UDPSender(
+            io_loop=channel.io_loop,
+            host=cfg.local_agent_reporting_host,
```

Comment:
here use the host port from local_agent_sender: https://github.com/jaegertracing/jaeger-client-python/blob/cb14ed80b699a48947d229b7d2c3f0917c32cac8/crossdock/server/endtoend.py#L105

Reply:
oh gawd it was the problem! I was really struggling.Thanks!